src/common/src/array/data_chunk.rs

```diff
@@ -344,18 +344,25 @@ impl DataChunk {
         Ok(outputs)
     }
 
-    /// Compute hash values for each row.
+    /// Compute hash values for each row. The number of the returning `HashCodes` is `self.capacity()`.
+    /// When `skip_invisible_row` is true, the `HashCode` for the invisible rows is arbitrary.
     pub fn get_hash_values<H: BuildHasher>(
         &self,
         column_idxes: &[usize],
         hasher_builder: H,
+        skip_invisible_row: bool,
```

Comment:
Better to make it a generic parameter

Reply:
Then should we also create a new function like `Array::hash_vec_with_bitmap` to static dispatch these two cases?